lib/ndr_import/file/xml.rb

```diff
@@ -14,21 +14,30 @@ class Xml < Base
 
       private
 
+      # Iterate through the file, yielding each 'xml_record_xpath' element in turn.
```

Comment:
Duplicate comment row, should be removed.

Reply:
Oops! Thanks @bshand - see 9226372